third_party/blink/renderer/platform/peerconnection/rtc_video_encoder_factory.cc

```diff
@@ -39,7 +39,8 @@ absl::optional<media::VideoCodecProfile> WebRTCFormatToCodecProfile(
       return absl::nullopt;
 #endif
 
-    return media::VideoCodecProfile::H264PROFILE_MIN;
+    return media::VideoCodecProfile::
+        H264PROFILE_HIGH444PREDICTIVEPROFILE;  // media::VideoCodecProfile::H264PROFILE_MIN;
```

Comment:
please revert commented code. and have this return statement on the single line of code

Reply:
This is an oversight from testing. Also, we use clang-format with the settings from the project, so formatting should be right.